src/pymoca/ast.py

```diff
@@ -293,99 +318,115 @@ def __init__(self, **kwargs):
 
 
 class IfStatement(Node):
     def __init__(self, **kwargs):
         self.conditions = []  # type: List[Union[Expression, Primary, ComponentRef]]
         self.blocks = []  # type: List[List[Union[AssignmentStatement, IfStatement, ForStatement]]]
-        self.comment = ''  # type: str
+        self.comment = ""  # type: str
         super().__init__(**kwargs)
 
     def __repr__(self):
-        return '{}(conditions={!r}, blocks={!r})'.format(
-            type(self).__name__, self.conditions, self.blocks)
+        return "{}(conditions={!r}, blocks={!r})".format(
+            type(self).__name__, self.conditions, self.blocks
+        )
 
 
 class WhenStatement(Node):
     def __init__(self, **kwargs):
         self.conditions = []  # type: List[Union[Expression, Primary, ComponentRef]]
         self.blocks = []  # type: List[List[Union[AssignmentStatement, IfStatement, ForStatement]]]
-        self.comment = ''  # type: str
+        self.comment = ""  # type: str
         super().__init__(**kwargs)
 
     def __repr__(self):
-        return '{}(conditions={!r}, blocks={!r})'.format(
-            type(self).__name__, self.conditions, self.blocks)
+        return "{}(conditions={!r}, blocks={!r})".format(
+            type(self).__name__, self.conditions, self.blocks
+        )
 
 
 class ForStatement(Node):
     def __init__(self, **kwargs):
         self.indices = []  # type: List[ForIndex]
         self.statements = []  # type: List[Union[AssignmentStatement, IfStatement, ForStatement]]
-        self.comment = ''  # type: str
+        self.comment = ""  # type: str
         super().__init__(**kwargs)
 
     def __repr__(self):
-        return '{}(indices={!r}, statements={!r})'.format(
-            type(self).__name__, self.indices, self.statements)
+        return "{}(indices={!r}, statements={!r})".format(
+            type(self).__name__, self.indices, self.statements
+        )
 
 
 class Function(Node):
     def __init__(self, **kwargs):
-        self.name = '' # type: str
+        self.name = ""  # type: str
         self.arguments = []  # type: List[Union[Expression, Primary, ComponentRef, Array]]
-        self.comment = ''  # type: str
+        self.comment = ""  # type: str
         super().__init__(**kwargs)
 
     def __repr__(self):
-        return '{}(name={!r}, arguments={!r})'.format(
-            type(self).__name__, self.name, self.arguments)
+        return "{}(name={!r}, arguments={!r})".format(
+            type(self).__name__, self.name, self.arguments
+        )
 
 
 class Symbol(Node):
     """
     A mathematical variable or state of the model
     """
-    ATTRIBUTES = ['value', 'min', 'max', 'start', 'fixed', 'nominal', 'unit', 'quantity', 'displayUnit']
+
+    ATTRIBUTES = [
+        "value",
+        "min",
+        "max",
+        "start",
+        "fixed",
+        "nominal",
+        "unit",
+        "quantity",
+        "displayUnit",
+    ]
```

Comment:
While vertically challenged, this does make it a little easier to peruse with diff tools that don't do character-by-character.